src/aap_eda/wsapi/routes.py

```diff
@@ -1,15 +1,28 @@
 from channels.routing import URLRouter
 from django.conf import settings
-from django.urls import path
+from django.urls import path, re_path
 
 from . import consumers
 
+default_path = re_path(r".*/?$", consumers.DefaultConsumer.as_asgi())
+
+
+default_router = URLRouter(
+    [
+        default_path,
+    ],
+)
+
 wsapi_router = URLRouter(
-    [path("ansible-rulebook", consumers.AnsibleRulebookConsumer.as_asgi())]
+    [
+        path("ansible-rulebook", consumers.AnsibleRulebookConsumer.as_asgi()),
```

Comment:
@Alex-Izquierdo @bzwei has a PR which will put authentication on this path https://github.com/ansible/eda-server/pull/540